internal/jobcontainers/jobcontainer.go

```diff
@@ -97,8 +98,12 @@ func newJobContainer(id string, s *specs.Spec) *JobContainer {
 	}
 }
 
+type JobContainerCreateOptions struct {
```

Comment:
Since we are already in `jobcontainers` package, prefer calling this just `CreateOptions`.